docs/getting-started.md

```diff
@@ -25,13 +25,90 @@ This diagram illustrates the structure of Luigi:
 
 <img src="https://github.com/SAP/luigi/blob/master/docs/assets/luigi-overview-diagram.jpg?raw=true" width="600"/>
 
+ Do you want to try out Luigi without installing it on your machine? Try one of these options:
+
+<!-- accordion:start -->
+
+ ### HTML file
+
+It is possible to create a Luigi application using a single HTML file. This example aims to show the simplicity of Luigi, but it is not intended for real-life use.
+
+Copy and paste this code in a text editor, then save it as an HTML file:
+
+```html
+ <!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<title>Hello Luigi</title>
+	<link rel='stylesheet' href='https://unpkg.com/@kyma-project/luigi-core/luigi.css'>
```

Comment:
If I am super picky, i would say the intendation of titel and link rel is wrong :-)